tests/testthat/test-find_bad_apples.R

```diff
@@ -1,3 +1,96 @@
-test_that("multiplication works", {
-  expect_equal(2 * 2, 4)
+# Makes dataframe for testing 'warning is raised for non-numerical data'
```

Comment:
@SamEdwardes You can disregard any notes you saw before about these tests not working. The changes I just made fix all the errors. The tests now pass! :)